IdentitySdkCore/IdentitySdkCore/Classes/ReachFiveApi.swift

```diff
@@ -64,15 +64,15 @@ public class ReachFiveApi {
     }
     
     public func clientConfig() -> Future<ClientConfigResponse, ReachFiveError> {
-        return Alamofire
+        return AF
```

Comment:
How did you create this alias? I don't find any reference in the PR.

Reply:
I didn't. It's the global default `Session` object in Alamofire 5.x.